config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml

```diff
@@ -11,7 +11,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: read-grafana
 rules:
-- apiGroups: ['grafana.integreatly.org']
+- apiGroups: ['grafanas.grafana.integreatly.org']
```

Comment:
Should't the api group be `grafana.integreatly.org` and the resource is `grafanas`

Reply:
Yes, it is leftover from my tries :)

Reply:
looks like I do not have the rights to resolve the comment thread I created... You will have to do it for me :)